scripts/build.sh

```diff
@@ -330,6 +342,66 @@ build_appletvos() {
    rm -rf ${TMP_BUILD_DIR}
 }
 
+build_watchos() {
+   local TMP_BUILD_DIR=$( mktemp -d )
+
+   # Clean up whatever was left from our previous build
+   rm -rf "${SCRIPT_DIR}"/../{watchsimulator/include,watchsimulator/lib}
+   mkdir -p "${SCRIPT_DIR}"/../{watchsimulator/include,watchsimulator/lib}
+
+   build "i386" "watchSimulator" ${TMP_BUILD_DIR} "watchsimulator"
+   build "x86_64" "watchSimulator" ${TMP_BUILD_DIR} "watchsimulator"
+   build "arm64" "watchSimulator" ${TMP_BUILD_DIR} "watchsimulator"
+
+   # The World is not ready for arm64e!
+   # build "arm64e" "AppleTVSimulator" ${TMP_BUILD_DIR} "watchsimulator"
+
+   rm -rf "${SCRIPT_DIR}"/../{watchos/include,watchos/lib}
+   mkdir -p "${SCRIPT_DIR}"/../{watchos/include,watchos/lib}
+
+   build "arm64" "watchOS" ${TMP_BUILD_DIR} "watchos"
+   build "arm64_32" "watchOS" ${TMP_BUILD_DIR} "watchos"
+   build "armv7k" "watchOS" ${TMP_BUILD_DIR} "watchos"
```

Comment:
It build but only arm64 headers are copied (below). I don't think `arm64_32` and `arm7k` is actually usable with this setup.